client/landing/stepper/declarative-flow/link-in-bio.ts

```diff
@@ -32,15 +33,17 @@ export const linkInBio: Flow = {
 		setStepProgress( flowProgress );
 		const siteSlug = useSiteSlug();
 		const userIsLoggedIn = useSelect( ( select ) => select( USER_STORE ).isCurrentUserLoggedIn() );
+		const urlLocale = useLocale();
 
 		function submit( providedDependencies: ProvidedDependencies = {} ) {
+			const localeRoute = urlLocale ? `/${ urlLocale }/` : '';
```

Comment:
We don't need to pass `en` when it's English.
```suggestion
			const localeRoute = urlLocale && urlLocale !== 'en' ? `/${ urlLocale }/`
```